recipes_source/profile_with_itt.rst

```diff
@@ -0,0 +1,144 @@
+
+With a successful profiling with ITT, you can open `Platform` tab of the profiling result to see labels in Intel® VTune™ Profiler timeline. All operators starting with `aten::` are operators labeled implicitly by the ITT feature in PyTorch. Labels `iteration_N` are explicitly labeled with specific APIs `torch.profiler.itt.range_push()`, `torch.profiler.itt.range_pop()` or `torch.profiler.itt.range()` scope. Please check the sample code in next section for details.
+
+.. figure:: /_static/img/itt_tutorial/vtune_timeline.png
+   :width: 100%
+   :align: center
+
+A short sample code showcasing how to use PyTorch ITT APIs
+----------------------------------------------------------
+
+Sample code below is the script that was used for profiling in the screenshots above.
+
+The topology is formed by 2 operators, `Conv2d` and `Linear`. Three iterations of inference were performed. Each iteration was labled by PyTorch ITT APIs as text string `iteration_N`. Either pair of `torch.profile.itt.range_push` and `torch.profile.itt.range_pop` or `torch.profile.itt.range` scope does the customized labeling feature.
+
+.. code:: python3
+
+   # sample.py
+
+   import torch
+   import torch.nn as nn
+   
+   class ITTSample(nn.Module):
+     def __init__(self):
+       super(ITTSample, self).__init__()
+       self.conv = nn.Conv2d(3, 5, 3)
+       self.linear = nn.Linear(292820, 1000)
+   
+     def forward(self, x):
+       x = self.conv(x)
+       x = x.view(x.shape[0], -1)
+       x = self.linear(x)
+       return x
+   
+   def main():
+     m = ITTSample()
+     x = torch.rand(10, 3, 244, 244)
+     with torch.autograd.profiler.emit_itt():
+       for i in range(3)
+         # Labeling a region with pair of range_push and range_pop
+         #torch.profiler.itt.range_push(f'iteration_{i}')
+         #m(x)
+         #torch.profiler.itt.range_pop()
+   
+         # Labeling a region with range scope
+         with torch.profiler.itt.range(f'iteration_{i}'):
+           m(x)
+   
+   if __name__ == '__main__':
+     main()
+
+
+The `launch.sh` bash script to wrap all manual steps is shown below.
+
+.. code:: bash
+
+   # launch.sh
+
+   #!/bin/bash
+   
+   # Retrive the directory path where contains both the sample.py and launch.sh so that this script can be invoked from any directory
```

Comment:
```suggestion
   # Retrive the directory path where the path contains both the sample.py and launch.sh so that this script can be invoked from any directory
```